app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> None:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
You can avoid using the `else` statement here. According to the checklist, it's better to write the code without the `else` statement if it's not necessary. You can just write `return 0` without the `else` statement because if the `if` condition is not met, the function will automatically proceed to the next line and execute it.